react-charts/stack-chart/step4.md

```diff
@@ -1,65 +1,35 @@
-Now we'll add multiple datasets to the `Chart` component...
+Add tooltips to the stack chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
-&lt;ChartBar 
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 1 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 2 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 5 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 3 }
-  ]} 
-/&gt;
+&lt;Chart
+  domainPadding={{ x: [30, 25] }}
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) <strong>Add a `containerComponent` property to the `Chart` component.</strong>

```

Comment:
Need to break this step up into multiple steps, like how its done in the Area Chart - step 4